ttnn/ttnn/decorators.py

```diff
@@ -325,7 +326,23 @@ def __hash__(self):
     def __call__(self, *function_args, **function_kwargs):
         return self.function(*function_args, **function_kwargs)
 
-    __doc__ = property(lambda self: self.function.__doc__)
+    def __post_init__(self):
+        if self.function.__doc__ is None:
+            return
+
+        # Delete the signature line created by pybind11
+        docstring_lines = self.function.__doc__.split("\n")
+        op_name = self.python_fully_qualified_name.split(".")[-1]
+        if f"{op_name}(" in docstring_lines[0]:
+            docstring_lines.pop(0)
+        self.__doc__ = "\n".join(docstring_lines)
+
+        # # TEMP HACK to read docstring from the file
```

Comment:
Is there a reason to keep this if it's commented out?
If yes, please say what it is as part of the comment.

Reply:
This is temporary, will be removed before the merge.